pysatNASA/instruments/timed_tidi.py

```diff
@@ -0,0 +1,255 @@
+fname_ncar = ''.join(('timed_windvectorsncar_tidi_',
+                      '{year:04d}{month:02d}{day:02d}',
+                      '????_v??.cdf'))
+supported_tags = {'': {tag: fname.format(ext=fname_ext[tag], gz='')
+                       for tag in tags},
+                  'ncar': {'vector': fname_ncar}}
+list_files = functools.partial(mm_gen.list_files,
+                               supported_tags=supported_tags,)
+
+# Set the load routine
+def load(fnames, tag='', inst_id=''):
+    """Load TIMED TIDI data into `xarray.DataSet` and `pysat.Meta` objects.
+
+    This routine is called as needed by pysat. It is not intended
+    for direct user interaction.
+
+    Parameters
+    ----------
+    fnames : array-like
+        iterable of filename strings, full path, to data files to be loaded.
+        This input is nominally provided by pysat itself.
+    tag : str
+        tag name used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself.
+    inst_id : str
+        Satellite ID used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself.
+
+    Returns
+    -------
+    data : xr.DataSet
+        A xarray DataSet with data prepared for the pysat.Instrument
+    meta : pysat.Meta
+        Metadata formatted for a pysat.Instrument object.
+
+    Raises
+    ------
+    ValueError
+        If temporal dimensions are not consistent
+
+    Note
+    ----
+    Any additional keyword arguments passed to pysat.Instrument
+    upon instantiation are passed along to this routine.
+
+    Examples
+    --------
+    ::
+
+        inst = pysat.Instrument('timed', 'tidi',
+                                inst_id='', tag='vector')
+        inst.load(2005, 179)
+
+    """
+
+    labels = {'units': ('Units', str), 'name': ('Long_Name', str),
+              'notes': ('Var_Notes', str), 'desc': ('CatDesc', str),
+              'plot': ('plot', str), 'axis': ('axis', str),
+              'scale': ('scale', str),
+              'min_val': ('Valid_Min', np.float64),
+              'max_val': ('Valid_Max', np.float64),
+              'fill_val': ('fill', np.float64)}
+
+    # Generate custom meta translation table. When left unspecified the default
+    # table handles the multiple values for fill. We must recreate that
+    # functionality in our table. The targets for meta_translation should
+    # map to values in `labels` above.
+    meta_translation = {'FIELDNAM': 'plot', 'LABLAXIS': 'axis',
+                        'ScaleTyp': 'scale', 'VALIDMIN': 'Valid_Min',
+                        'Valid_Min': 'Valid_Min', 'VALIDMAX': 'Valid_Max',
+                        'Valid_Max': 'Valid_Max', '_FillValue': 'fill',
+                        'FillVal': 'fill', 'TIME_BASE': 'time_base'}
+    if inst_id == 'ncar':
+        if tag == 'vector':
+            data, meta = cdw.load(fnames, tag, inst_id,
+                                  pandas_format=True)
```

Comment:
```suggestion
                                  pandas_format=True,
                                  meta_translation=meta_translation)
```
@jklenzing I think we may need a meta_kwargs pass thru in cdw.load to supply the labels dict to meta.

meta_translation is also not currently supported by the pandas_format option. My suggestion may be premature.

Reply:
Example of metadata from this data set
```
In [39]: tidi.meta['Epochcold']
Out[39]: 
FIELDNAM              Epoch cold
VAR_TYPE            support_data
DEPEND_0                     NaN
DEPEND_1                     NaN
LABL_PTR_1                   NaN
DISPLAY_TYPE                 NaN
FORMAT                       NaN
LABLAXIS                     NaN
SCALETYP                     NaN
units                        NaN
long_name              Epochcold
notes                  Epochcold
desc                Default time
value_min       62798371200000.0
value_max       63460972800000.0
fill                         NaN
Name: Epochcold, dtype: object
```

Reply:
This could be improved. I used a generic metadata dict and not sure how they match with the fields within the loaded TIDI files. I would need assistance for that.

Reply:
No problem. Posted this to help with our pysat meeting conversation on the code. Thanks again for the pull!